mlir/test/Dialect/SPIRV/IR/func-entry.mlir

```diff
@@ -0,0 +1,12 @@
+// Bug: https://github.com/llvm/llvm-project/issues/132894
+
+// RUN: not mlir-opt %s 2>&1 | FileCheck %s
+
+// CHECK: error: 'spirv.func' op entry block must have 1 arguments to match function signature
+module {
+  spirv.func @f(f32) "None" {
+    %c0 = arith.constant 0 : index
+    spirv.Return
+  }
+}
```

Comment:
I suspect this test should go to `function-decorations.mlir`

Also look at the last test in the file to see how we check for errors, this is not using FileCheck CHECK but the verify-diagnostics mechanism.